Gopkg.toml

```diff
@@ -68,23 +68,47 @@
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/helm"
-  version = "2.12.0"
+  version = "2.14.3"
+
+[[override]]
+  name = "k8s.io/cli-runtime"
+  version = "kubernetes-1.15.4"
```

Comment:
Are these overrides permanent or can we remove once other PRs are merged?

Reply:
These are permanent, until dep improves to work with Go modules metadata (used by k8s libraries) or we are able to switch to Go modules (currently that's not possible since k8s library release tags are not following semver).

Without these transitive dependency overrides `dep ensure -update` doesn't complete resolving dependencies (not as "fast" as Linux which "can do an infinite loop in under 5 seconds") or resolves wrong (typically master) version/branch.